pandas/core/accessor.py

```diff
@@ -12,7 +12,8 @@
 
 class DirNamesMixin(object):
     _accessors = frozenset([])
-    _deprecations = frozenset(['asobject'])
+    _deprecations = frozenset(
+        ['asobject', 'base', 'data', 'flags', 'itemsize', 'strides'])
```

Comment:
BTW, this only adds them to Index, not to Series. 
So not sure why you did get warnings for Index, but not for Series in the first place.